src/TeamsConnector.php

```diff
@@ -18,18 +18,20 @@ public function __construct($webhookUrl)
      * Sends card message as POST request
      *
      * @param  TeamsConnectorInterface $card
+     * @param  int $curlOptTimeout by default = 10
+     * @param  int $curlOptConnectTimeout by default = 3
      * @throws Exception
      */
-    public function send(TeamsConnectorInterface $card)
+    public function send(TeamsConnectorInterface $card, int $curlOptTimeout = 10, int $curlOptConnectTimeout = 3)
```

Comment:
the type hinting for int is only supported by PHP7.0+ so we need to remove them for now

Reply:
I have updated the code